awx/main/tests/functional/tasks/test_tasks_system.py

```
@@ -105,6 +112,82 @@ def test_folder_cleanup_multiple_running_jobs(job_folder_factory, me_inst):
+        from awx.main.tasks.system import delete_inventory
+
+        inv = self._make_inventory_with_hosts(3)
+        inv_id = inv.id
+
+        call_count = {'n': 0}
+        original = _batched_delete_inventory.__wrapped__ if hasattr(_batched_delete_inventory, '__wrapped__') else _batched_delete_inventory
+
+        def flaky_delete(inventory, batch_size=500):
+            call_count['n'] += 1
+            if call_count['n'] == 1:
+                raise DatabaseError('connection reset')
```

Ruff (0.15.21)

[error] Undefined name `DatabaseError`

(F821)